test/phoenix/router/routing_test.exs

```diff
@@ -34,6 +39,8 @@ defmodule Phoenix.Router.RoutingTest do
     match :move, "/move", UserController, :move
 
     get "/users/:user_id/files/:id", UserController, :image
+    get "/standard", StandardPlug
```

Comment:
let's use a function plug here instead so we don't have to create another module